src/UserSettingsPage.cpp

```diff
@@ -93,15 +93,15 @@ UserSettings::load(std::optional<QString> profile)
         sortByImportance_     = settings.value("user/sort_by_unread", true).toBool();
         readReceipts_         = settings.value("user/read_receipts", true).toBool();
         theme_                = settings.value("user/theme", defaultTheme_).toString();
-        font_                 = settings.value("user/font_family", "default").toString();
+        font_                 = settings.value("user/font_family", tr("Default")).toString();
```

Comment:
I think that will cause issues. You should translate that to default in the setFont functions and such instead, I think